### ISSUE TITLE
Configure PSR12.ControlStructures.BooleanOperatorPlacement to only allow start of line operator placement

### ISSUE THREAD

**Issue by Hornet-Wing** [squizlabs/PHP_CodeSniffer#2743]
Would it be possible to add an option to specify where the boolean operator should be placed in a multi-line control structure?

My reasoning:

Since the addition of `PSR12.ControlStructures.ControlStructureSpacing.FirstExpressionLine` we've switched from `PEAR.ControlStructures.MultiLineCondition.SpacingAfterOpenBrace`. However, one of the things we're missing is a sniff for the operator placement (start or end of statement) which was previously provided by the `PEAR.ControlStructures.MultiLineCondition.StartWithBoolean` sniff. Unfortunately, I am unable use the PEAR sniff as it also flags the first line of the control structure as missing an operator (as shown in example 3).

Examples:

1. My current set up using `PSR12.ControlStructures.ControlStructureSpacing.FirstExpressionLine` and `PSR12.ControlStructures.BooleanOperatorPlacement`. I'd like this to error as the `&&` is at the end of the statement.
```php
if (
	$one === $two &&
	$someValue
) {
	echo 'ok';
}
```
```
No errors.
```

2. Using `PEAR.ControlStructures.MultiLineCondition`, this is the previous behaviour we had using the PEAR sniffs.
```php
if ($one === $two &&
	$someValue
) {
	echo 'ok';
}
```
```
---------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------------------------
 2 | ERROR | [x] Each line in a multi-line IF statement must begin with a boolean operator
   |       |     (PEAR.ControlStructures.MultiLineCondition.StartWithBoolean)
---------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
---------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 616ms; Memory: 14MB
```

3. Using a bit of both to get the operator position errors. (`PSR12.ControlStructures.ControlStructureSpacing.FirstExpressionLine` and `PEAR.ControlStructures.MultiLineCondition.StartWithBoolean`). This output makes sense as PEAR expects the first line to be on a new line.
```php
if (
	$one === $two
	&& $someValue
) {
	echo 'ok';
}
```
```
-----------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------------------------------------------
 2 | ERROR   | [x] Each line in a multi-line IF statement must begin with a boolean operator
   |         |     (PEAR.ControlStructures.MultiLineCondition.StartWithBoolean)
-----------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 1.49 secs; Memory: 16MB
```

Does this make sense?

**Comment by jrfnl** [comment]
Duplicate of #2620 ? Also see PR #2680.

**Comment by gsherwood** [comment]
This feature is scheduled for 3.5.4. Closing as duplicate.
